### PR TITLE
Proper manage connection

### PR DESCRIPTION
**Pull request by nguyenkims**

### CODE REVIEW

File: migration_sql/db.py [hunk 1]
```diff
@@ -35,24 +34,48 @@ def __init__(self, host, port, username, password, name):
...
+
+        if not self._exist_table("schema_version"):
+            log.debug("schema_version not exist, create table schema_version")
+            self._create_schema_version_table()
 
     @contextmanager
     def get_session(self):
         """
-        to make sure to close the session
+        Get the SQLAlchemy session to the database and close the connections after.
```
Comment (hxuanhung) [inline migration_sql/db.py]: I think this sentence is not quite clear. You mean create new sesssion and roll back if not succeeded?

File: migration_sql/db.py [hunk 2]
```diff
@@ -3,15 +3,14 @@
...
 from sqlalchemy import create_engine
-from sqlalchemy.orm import sessionmaker
+from sqlalchemy.orm import sessionmaker, scoped_session
 
-from . import schema_version
 from .schema_version import SchemaVersion
 from .version import Version
 
-l = logging.getLogger("ww.migration")
+log = logging.getLogger("ww.migration")
```
Comment (hxuanhung) [inline migration_sql/db.py]: (this is not important) I see in API, we use `logger` instead of `log`.

File: migration_sql/db.py (hunk 1 above)
Comment (nguyenkims) [reply migration_sql/db.py]: this one implements the `with as` pattern in python to make sure the connection is closed afterwards.

File: migration_sql/db.py (hunk 2 above)
Comment (nguyenkims) [reply migration_sql/db.py]: yeah it's the same, I prefer `log` as it's shorter to type :)

**Pull request merged**